### DISCUSSION TITLE
What does the Reference Folders button do?

### DISCUSSION THREAD

**Discussion by AlphaHasher** [qarmin/czkawka#680] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/33443001/163941556-340088e4-044e-4f0d-ad65-8d42662d45bf.png)

**Comment by qarmin** [comment, marked as answer]
Referenced folder means something like original.

Files in this folder are treated as original files which cannot be selected/removed/moved etc.

Only results from non referenced folders are visible under first result from referenced folder and can be freely modified

**Reply by AlphaHasher** [reply]
Oh, so if for example I have two folders and I only want to manipulate the second folder but still check it against the first one I would have to click the reference folder button on the first folder

**Comment by ghost** [comment]
@qarmin  Can you please add a tooltip when hovering over the words 'reference folders' and/or the checkbox? This feature is great, but it is not intuitive or  seemingly explained anywhere but here.

**Reply by xeruf** [reply]
plus highlighting in the duplicates view e.g. greying it out - right now it only becomes apparent which one is in the reference when you click on them which was a bit unintuitive for me
